src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -245,6 +598,14 @@ public void unsupportedBinaryContentHandlingAttribute(final String uri) {
                         "to fail must be described in a resource indicated by a " +
                         "rel=\"http://www.w3.org/ns/ldp#constrainedBy\" link in the Link response header.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUri, "unsupported")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.